globus_folder_sync.py

```diff
@@ -15,14 +15,15 @@
 
 import json
 import sys
-import webbrowser
 import os
 import six
 
 from globus_sdk import (NativeAppAuthClient, TransferClient,
                         RefreshTokenAuthorizer, TransferData)
 from globus_sdk.exc import GlobusAPIError, TransferAPIError
 
+from native_login import NativeClient
```

Comment:
One minor change needed here. Shortly, I'll change the name of the `native-login` package to `fair-research-login`, ([PR is here](https://github.com/fair-research/native-login/pull/16)) so this line will need to change to:

    from fair_research_login import NativeClient

The line in the `requirements.txt` will need a change too. The repo's name will stay the same, so that won't need to change. Hopefully soon, the package will be on pypi so the install can change to a simple:

    pip install fair-research-login